test/unit/vdom/incremental-dom.js

```diff
@@ -32,17 +32,17 @@ describe('IncrementalDOM', () => {
 
       patchIt('elementOpen, elementClose', () => {
         vdom.elementOpen(Elem);
-        expect(vdom.elementClose(Elem).outerHTML).to.equal('<x-test></x-test>');
+        expect(vdom.elementClose(Elem).tagName).to.equal('X-TEST');
```

Comment:
These tests had to be updated because they were testing `outerHTML`. The result is different depending on the browser so it's safer to test `tagName` as it's not producing any content anyways.